src/helpers.js

```diff
@@ -105,6 +125,11 @@ const getPrettierOptions = (editor: TextEditor) => ({
   jsxBracketSameLine: getPrettierOption('jsxBracketSameLine'),
 });
 
+const getPrettierCliOptions = (options: Object) =>
+  Object.keys(options)
+    .reduce((cliOptions, option) => [...cliOptions, `${CLI_OPTIONS[option]} ${options[option]}`], [])
+    .join(' ');
```

Comment:
It seems like you're using a `reduce` to imitate a `map`? I think you could just use a map directly:

```js
const getPrettierCliOptions = (prettierOptions: {}) => 
 Object.keys(prettierOptions).map(option => `${CLI_OPTIONS[option]} ${prettierOptions[option]`).join(' ');
```

or you could stay with reduce and skip the array/join:

```js
const getPrettierCliOptions = (prettierOptions: {}) => 
 Object.keys(prettierOptions).reduce((acc, option) => `${acc} ${CLI_OPTIONS[option]} ${prettierOptions[option]`);
```

Also, I think we maybe should just take the `editor` as an argument and call `prettierOptions` ourselves instead of making the caller do it, that way the caller is less coupled:

```js
const getPrettierCliOptions = (editor: TextEditor) => 
 Object.keys(getPrettierOptions(editor)).reduce((acc, option) => `${acc} ${CLI_OPTIONS[option]} ${prettierOptions[option]`);
```